lib/rules/require-render-return.js

```diff
@@ -20,15 +20,15 @@ function create (context) {
     utils.executeOnVue(context, obj => {
       const node = obj.properties.find(item => item.type === 'Property' &&
         utils.getStaticPropertyName(item) === 'render' &&
-        (item.value.type === 'ArrowFunctionExpression' || item.value.type === 'FunctionExpression') &&
-        !item.value.expression // render: () => test
+        (item.value.type === 'ArrowFunctionExpression' || item.value.type === 'FunctionExpression')
       )
       if (!node) return
 
       forbiddenNodes.forEach(el => {
         if (
           el.loc.start.line >= node.value.loc.start.line &&
-          el.loc.end.line <= node.value.loc.end.line
+          el.loc.end.line <= node.value.loc.end.line &&
+          el.loc.start.line === node.value.loc.start.line
```

Comment:
I meat we can remove all other conditions and leave only this one. Or even just leave:
```
node.value === el
```
:D up to you